Documentation/Contributing/development-flow.md

```diff
@@ -77,6 +77,11 @@ git remote -v
 
 Now you should have at least `origin` and `upstream` remotes. You can also add other remotes to collaborate with other contributors.
 
+### Self assign Issue
+
+You can self-assign any issue that is been created in Rook upstream repo, by adding
+an issue comment with `/assign` in the body.
```

Comment:
```suggestion
You can self-assign any issue that is open and not assigned to anyone in Rook upstream repo, by adding a comment with `/assign` in the body.
```

Reply:
Done changed the `not assigned` thing that will be really helpful

But `an issue comment` seems fine and more understandable so keep like that, 
let me if it doesn't look good.